torch/utils/data/dataloader.py

```diff
@@ -119,6 +117,13 @@ def _sharding_worker_init_fn(worker_init_fn, world_size, rank_id, worker_id):
         worker_init_fn(worker_id)
 
 
+def _share_dist_seed(generator, pg):
```

Comment:
Question: so this handles everything that `_get_shared_seed` did? Including `seed_counter`, `timeout`, `check_interval`?

Reply:
Yep. ProcessGroup itself contains `timeout` and `check_interval`. And, the counter would rely on `broadcast` function.